sdk/iot/Azure.Iot.Hub.Service/src/API Design.md

```diff
@@ -71,89 +70,178 @@ public class Devices
     /// <returns>The retrieved device.</returns>
     public virtual async Task<Response<DeviceIdentity>> GetIdentityAsync(string deviceId, CancellationToken cancellationToken = default)
 
+    /// <summary>
+    /// Get a single device identity.
+    /// </summary>
+    /// <param name="deviceId">The unique identifier of the device to get.</param>
+    /// <param name="cancellationToken">The cancellation token.</param>
+    /// <returns>The retrieved device.</returns>
+    public virtual Response<DeviceIdentity> GetIdentity(string deviceId, CancellationToken cancellationToken = default)
+
+    /// <summary>
+    /// Delete a single device identity.
+    /// </summary>
+    /// <param name="deviceIdentity">The device to delete. If no ETag is present on the device, then the condition must be equal to <see cref="IfMatchPrecondition.Unconditional"/> or equal to <see cref="IfMatchPrecondition.UnconditionalIfMatch"/></param>
```

Comment:
What is the behavior if `DeviceIdentity.Etag = null` and `precondition = IfMatchPrecondition.IfMatch`?

Reply:
Then a null ETag will be given to the PL, and the PL has logic to not send an If-Match header if the ETag is null. Effectively it is the same as just doing ```precondition = unconditional```